providers/redfish/inventory.go

```diff
@@ -0,0 +1,673 @@
+package redfish
```

Comment:
This is a pretty long file, any way this can be logically broken up into more readable files?

Reply:
I've split out the collectors from the file in [da0793a](https://github.com/bmc-toolbox/bmclib/pull/267/commits/da0793ac66a6bd8dec55ecf696e376dd58bb90bc)